Expense/expense.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
We can use object destructuring here: `const { ObjectId } = mongoose.Schema.Types`